.github/workflows/engine-nightly.yml

```diff
@@ -42,6 +42,12 @@ jobs:
         run: ./run git-clean
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Setup Python
+        uses: actions/setup-python@v5
```

Comment:
Python is required to run the `npm install`

Reply:
Why? Are we installing some special Python requiring `npm` module?